doc/guide/rp2040-update.md

```diff
@@ -2,11 +2,11 @@
 
 Before plugging in your Sonata board, hold down the SW9 labelled "RP2040 Boot", and while holding this button plug your board into your laptop using the Main USB.
 
-A drive called RPI-RP2 should pop up on your computer and drag `rpi_rp2_v0.X.uf2` into it.
-This drive should automatically dismount once the file is transferred and remount as SONATA.
+A drive called 'RPI-RP2' should pop up on your computer and drag `rpi_rp2_vX.Y.uf2` into it.
+This drive should automatically dismount once the file is transferred and remount as 'SONATA'.
 
-## Downloading the `rpi_rp2_v.X.uf2` file
+## Downloading the `rpi_rp2_vX.Y.uf2` file
 
-Currently the RP2040 firmware is available from the [Sonata Systems](https://github.com/lowRISC/sonata-system/releases) releases, which ensures your RP2040 firmware matches the Sonata FPGA and firmware expectations.
+Currently the RP2040 firmware is available from the [Sonata Systems releases](https://github.com/lowRISC/sonata-system/releases), which ensures your RP2040 firmware matches the Sonata FPGA and firmware expectations.
```

Comment:
Perhaps `[Sonata System releases page]` would be clearer?

Reply:
Thanks, changed.